packages/mdc-textfield/adapter.js

```diff
@@ -106,6 +106,17 @@ class MDCTextFieldAdapter {
    */
   deregisterInputInteractionHandler(evtType, handler) {}
 
+  /**
+   * Registers a validation attribute change listener on the input element.
+   * @param {function(!Array): undefined} handler
+   */
+  registerValidationAttributeChangeHandler(handler) {}
+
+  /**
+   * Disconnect all validation attribute listeners on the input element.
+   */
+  deregisterValidationAttributeChangeHandler() {}
```

Comment:
This should probably just disconnect a specific observer that it is handed, and we should loop within the component's destroy method instead of building the iteration logic into the adapter.